### ISSUE TITLE
Legitimate site : descoteaux.top

### ISSUE THREAD

**Issue by deseri01** [DandelionSprout/adfilt#709]
## Which entry/entries are you submitting?

descoteaux.top

## Which things do they block, hide, or unbreak?

All domain / sub domain for descoteaux.top that I had purchased today

## Which of my lists are you submitting it to?
Dandelion Sprout's Anti-Malware List

## Which adblocker(s) and version did you use when writing and testing the entries?

- [ ] uBlock Origin
- [ ] AdGuard (Paid desktop version)
- [ ] AdGuard (Gratis browser versions that aren't Manifest V3 nor Safari)
- [ ] AdBlock (Firefox or Manifest V2)
- [ ] Adblock Plus (Firefox or Manifest V2)
-   AdGuard Home
- [ ] Blokada
- [ ] I Don't Care About Cookies (The extension)
- [ ] AdNauseam

Other(s):

Adblocker version(s):
[Adblock Plus 3.13]
! Title: 💊 Dandelion Sprout's Anti-Malware List (for AdGuard Home, and for AdGuard for Android/Windows' DNS filtering)
! Version: 07December2022v1

## Which filterlists did you use? Failing to tell this will temporarily close the report until it has been told.

||top^$denyallow=corriente.top|gdtot.top|nicenature.top|reminder.top|magocoro.top|castlevania.top|suiten.top|shucks.top|1stream.top|ambr.top|techblog.top|changlam10.top|changlam11.top|pdcdn1.top|mastodon.top|pressplay.top

**Comment by iam-py-test** [comment]
There's a whois record, but that domain returns NXDOMAIN. Until there is actually a DNS record and a website, I personally do not see a point in allowlisting it or any way to verify it would not be malicious.
For reference (not that it makes **any** difference as I doubt there has been enough time for security venders to catch any abuse): https://www.virustotal.com/gui/url/768c0ab0cc6c146a99af9b54904f2214e7dfcbc2e9ac0e4d67eb4e92ba570efa/detection (VT seems to be able to resolve it to an IP, I can't)

**Comment by DandelionSprout** [comment]
I'm inclined to agree with iam-py-test, in the sense that the turnover/re-sale/hijack rates of `.top` domains are very, very high. At the very least a site would need to demonstrate that it is used for legitimate purposes, e.g. to serve as an archive or filehost for a project.

In the meantime, you (deseri01) can add `@@||descoteaux.top^` to "Custom filter entries" in the AdGuard Home admin settings.

**Comment by deseri01** [comment]
Thanks @DandelionSprout ,  I've already added the exception in my AdGuard Home filter.  I was following instruction from https://www.youtube.com/watch?v=Q5dG8g4-Sx0&ab_channel=DBTech about using cloudflare tunneling to reach securely my docker's apps at home.  I have realized that domain with .top were blocked after I had purchased for 10 years ;( . I will correct the DNS entry in CloudFlare so it will be legitimate to add my exception on the list. 

Best regards,
